metadata/Cadastre/plss_quarter_quarter_sections.md

```diff
@@ -0,0 +1,75 @@
+# Title
+
+Utah PLSS Quarter Quarter Sections GCDB
+
+## ID
+
+4b63af59-da17-40dd-bf40-2996c489fc4d
+
+## Brief Summary
+
+Polygon dataset of the smallest unit of the Public Land Survey System (PLSS), where each unit represents 40 acres of land.
+
+## Summary
+
+This dataset represents the smallest unit of the Public Land Survey System (PLSS). Each polygon represents approximately 40 acres of land and is labelled by its township and range. We used Geographic Coordinate Database ([GCDB](https://www.sciencebase.gov/catalog/item/4f4e4a7fe4b07f02db648dce)) base attributes to create the second division class (quarter quarter sections).
+
+## Description
+
+### What is the dataset?
+
+Originally established in 1785 and currently administered by the Bureau of Land Management (BLM), the PLSS is a nationwide framework for surveying, dividing, and quantifying land. PLSS sections are divided into a grid with 640 acres in each square, which is further divided into quarter sections (160 acres per square) and quarter quarter sections (40 acres per square). This dataset represents this final division as polygons.
+
+The polygons in this feature class represent the smallest division that has been defined by the BLM. This data is derived from the [Utah PLSS Fabric](https://gis.utah.gov/products/sgid/cadastre/parcel-fabric/). Other PLSS datasets can be found in the [Cadastre Data Index](https://gis.utah.gov/products/sgid/cadastre/).
+
+### What is the purpose of the dataset?
+
+PLSS data are used in many areas of natural resource management, land ownership designation, and public planning. Though not a substitute for a legal land survey, this dataset is a helpful resource for cartography and general analytic purposes.
+
+### What does the dataset represent?
+
+This dataset represents PLSS quarter quarter sections as polygons spanning 40 acres each. The attributes assigned to PLSS polygons were taken from the BLM's Legal Land Description (LLD) dataset, which can be found in the the Legacy Rehost for the year 2000 (LR2000) automated records system.
+
+### How was the dataset created?
+
+We created this dataset using cadastral survey records housed by the BLM, local records, and geographic control coordinates from states, counties, and federal agencies. We dissolved individual lots that were smaller than 40 acres to the section level. This data was originally published in January of 2017 and is currently maintained by UGRC in partnership with the BLM and local government authorities.
+
+### How reliable and accurate is the dataset?
+
+This dataset represents the most current and complete version of PLSS quarter quarter sections in Utah. Divisions beyond the quarter quarter sections may be included in this feature class, but in most implementations these smaller divisions will be [parcels](https://gis.utah.gov/products/sgid/cadastre/parcels/).
```

Comment:
**thought**: does this contradict the above statements that these are the final and smallest divisions?

Reply:
I think it depends on whether this dataset still has divisions smaller than the quarter quarter ones. The original metadata included this disclaimer, but if those smaller divisions have since been removed from this feature class, then I don't think we need to mention it at all.

Reply:
There is a part of the cadnsdi that is known as special surveys within the plss intersected data that is a smaller division of the PLSS which is shown as lots and mining claims. https://gis.utah.gov/products/sgid/cadastre/cadnsdi/

Reply:
Gotcha. I'll adjust the metadata to mention the special surveys as the smallest division.